nginx/redirects.conf

```
@@ -8,3 +8,4 @@ rewrite ^/integration/ecommerce_platforms/demandware/?$ /integration/ecommerce_p
 rewrite ^/payment_solutions(?:/recharge)?/?$ /integration/custom/recharge/ permanent;
 rewrite ^/integration/custom_integration/?$ /integration/standard/ permanent;
 rewrite ^/custom_integration/(.*)$ /integration/custom/$1 permanent;
+rewrite ^/integration/ecommerce_platforms/magento/(magento1|magento2)/?$ /integration/ecommerce_platforms/magento/ permanent;
```

Please check if this works.
```suggestion
rewrite ^/integration/ecommerce_platforms/magento/magento[12]/?$ /integration/ecommerce_platforms/magento/ permanent;
```

---

@zhuravel yes it works, checked